### PR TITLE
Clarify how to contribute & role of collaborators

### PR DESCRIPTION
**Pull request by drsm79**
- [x] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](../blob/master/DCO1.1.txt)

## What

Clarify how to contribute to the project & the role of collaborators

## Why

Documentation doesn't map to how we want to work in the OSS world.

## How

_Provide a bulleted list of the changes included in the pull request._

## Test

_Provide a bulleted list of tests included in the pull request and/or tests
performed to validate the work included in the pull request._

## Context

_Provide a bulleted list of GitHub issues, or any other references (mailing list discussion, etc...) that reviewers can reference
for additional information regarding scope of the pull request._

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Contributing
 
-If you want to add to the framework, please familiarise yourself with the code & our [Coding Standards][]. Before you submit a PR, please [file an issue][new collab] to request collaborator access.
+If you want to add to the framework, please familiarise yourself with the code & our [Coding Standards][]. Make a fork of the repository & file a Pull Request from your fork with the changes. You will need to click the checkbox in the template to show you agree to the [Developer Certificate of Origin](../blob/master/DCO1.1.txt).
+
+If you make regular & substantial contributions to Auditree, you may want to become a collaborator. This means you can approve pull requests (though not your own) & create releases of the tool. Please [file an issue][new collab] to request collaborator access. A collaborator supports the project, ensuring coding standards are met & best practices are followed in contributed code, cutting & documenting releases, promoting the project etc.
```
Comment (alfinkel) [inline CONTRIBUTING.md:5]: Right now PRs can only be approved by CODEOWNERS are you looking to remove that and allow any collaborator to approve PRs?

Comment (drsm79) [reply CONTRIBUTING.md:5]: I think I'm saying CODEOWNERS & Contributors are synonymous now.

Comment (alfinkel) [reply CONTRIBUTING.md:5]: Can you bold **regular & substantial contributions**?

Comment (drsm79) [reply CONTRIBUTING.md:5]: sure

**Pull request merged**